test/e2e/network/dns.go

```diff
@@ -318,4 +318,8 @@ var _ = SIGDescribe("DNS", func() {
 
 		// TODO: Add more test cases for other DNSPolicies.
 	})
+
+	It("should forward PTR records to upstream server by kubedns", func() {
+
+	})
```

Comment:
where is the actual test ;-)

Reply:
This PR is not ready for review yet. It's a place holder.
I will work on this when the kubedns side is merged ([PR#220 in k8s/dns](https://github.com/kubernetes/dns/pull/220)) and will remove [WIP] when this one is ready. Sorry for confusion.